tests/integration/frameworks/drupal/skipif.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+if (version_compare(PHP_VERSION, "8.2", ">=")) {
```

Comment:
Is this related to the changes in this PR?

Reply:
WordPress and Drupal instrumentation is using code paths modified by this PR. There already existed WordPress integration tests. However there were no Drupal integration tests. This PR adds Drupal integration tests to increase coverage of the Drupal instrumentation and code paths modified by this PR.

Reply:
Is there a similar Drupal test for PHP >= 8.2?

Reply:
After discussion it was determined the test for PHP >= 8.2 is because the `invoke_all` Drupal API call is not available in Drupal versions which run on PHP >= 8.2.  The test using `invoke_all_with` is run for those PHP versions.

Reply:
Yes there is. Older version of Drupal (the one that does not run on PHPs 8.2+) use one method of invoking custom plugins and this method is tested with `tests/integration/frameworks/drupal/test_module_invoke_all.php`. Newer version of Drupal (the one that runs on PHPs 7.4+) use another method of invoking custom plugins and this method is tested with `tests/integration/frameworks/drupal/test_invoke_all_with.php`.